examples/00-fluent/modeling_ablation.py

```diff
@@ -34,7 +34,7 @@
 # the damaging effects of external high temperatures caused by shock wave and viscous
 # heating. The ablative material is chipped away due to surface reactions that remove a
 # significant amount of heat and keep the vehicle surface temperature below the melting
-# point. In this tutorial, Fluent ablation model is demonstrated for a re-entry vehicle
+# point. In this tutorial, Fluent ablation model is demonstrated for a reentry vehicle
```

Comment:
AFAICS, "re-entry" is the correct English language word, while "reentry" is used in the USA. I believe that we're forced to "prefer" the latter form.